actions/patch_node_handler.go

```diff
@@ -56,10 +56,6 @@ func (h *patchNodeHandler) Handle(ctx context.Context, action *castai.ClusterAct
 		"id":        action.ID,
 	})
 
-	// on GKE we noticed that sometimes the node is not found, even though it is in the cluster
```

Comment:
Why you meved that commend to function body?

Reply:
The comment I moved mentions retries but adjacent lines do not have retries because retry happens inside of  "get node" function. 
Would you agree that keeping the comment close the code it relates would be better?

That said I wouldn't have necessarily moved it but since I  put in the comment the real cause why client doesn't find new node right away (i.e. delayed cache sync) I decided to move it where it suits more.